src/test/java/decodes/cwms/TestCwmsTimeSeriesDAO.java

```diff
@@ -0,0 +1,40 @@
+package decodes.cwms;
+
+import java.util.Calendar;
+import java.util.TimeZone;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+final class TestCwmsTimeSeriesDAO
```

Comment:
.. wait, don't we usually put "test" at the end of the test class name?

Reply:
Yes, I believe that is the standard imposed by xUnit. I renamed